src/js/tests.rs

```diff
@@ -0,0 +1,28 @@
+mod strings;
+
+#[macro_export]
+macro_rules! assert_js {
```

Comment:
This is "mostly" copy pasted from erl.rs I think that's probably best than finding a way to reuse the logic but switch on target somehow.